buildSrc/build.gradle

```diff
@@ -117,7 +117,7 @@ dependencies {
   compile 'com.netflix.nebula:nebula-publishing-plugin:4.4.4'
   compile 'com.netflix.nebula:gradle-info-plugin:7.1.3'
   compile 'org.apache.rat:apache-rat:0.11'
-  compile "org.elasticsearch:jna:4.5.1"
+  compile "org.elasticsearch:jna:5.5.0"
```

Comment:
Why don't we use the `version.properties` version here? We load this file even for the `buildSrc` build so we can reference it here, as we de below for things like checkstyle and randomized runner. That would save us another place to bump this version in the future.

Reply:
I don't know why, can you pick this up in a follow-up?